src/Renderer/RasterTile.js

```diff
@@ -182,7 +190,6 @@ export class RasterElevationTile extends RasterTile {
             if (this.min != min || this.max != max) {
                 this.min = min;
                 this.max = max;
-                this.dispatchEvent({ type: 'updatedElevation', node: this });
```

Comment:
Maybe we should leave it, rename it to `minMaxElevationChanged`, send the min and max with it and export it? WDYT?

Reply:
I find it duplicates the other event (`rasterElevationChanged`) because the only time where the min/max changes is when the texture level changes

Reply:
ok